app/views/admin/edition_change_notes/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<% content_for :page_title, "#{@edition.title}: Change notes" %>
+<% content_for :context, @edition.title %>
+<% content_for :title, "Change notes" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/table", {
+      head: [
+        {
+          text: "Date",
+        },
+        {
+          text: "Change note",
+        },
+        {
+          text: "Actions",
+        },
+      ],
+      rows: @change_notes.map do |change_note|
+        [
+          {
+            text: I18n.l(change_note.major_change_published_at, format: :long_ordinal),
+          },
+          {
+            text: change_note.change_note,
+          },
+          {
+            text: render("govuk_publishing_components/components/button", {
+              text: "Edit",
```

Comment:
For other pages we've used separate links to an edit page and a confirm delete page as David mentioned.
https://github.com/alphagov/whitehall/pull/7083/files#diff-dea9c5138c955d5e779f2c0776692f5063ef6535ebb4afdefeb7ddacdbf99b21R284